openapi/openapi_public.yaml

```diff
@@ -16,6 +16,7 @@ info:
     * Citation helper: https://github.com/adsabs/citation_helper_service
     * Classic import: https://github.com/adsabs/harbour-service
     * Objects: https://github.com/adsabs/object_service
+    * Journals: https://github.com/adsabs/ADSJournalsDB
```

Comment:
Should point to the service repo, not the pipeline, since that's where the endpoints you're documenting are. 

Also, I noticed that the journals service repo is private - can we make this public?

Reply:
The service is located in ADSJournalsDB/journalsservice -- this was per RC's recommendation.